server/utils/CacheHelper.ts

```diff
@@ -7,7 +7,7 @@ import Redis from "@server/storage/redis";
  */
 export class CacheHelper {
   // Default expiry time for cache data in seconds
-  private static defaultDataExpiry = Day / Second;
+  private static defaultDataExpiry = Day / 1000;
```

Comment:
Changed this since reading `Day / Second` conveys that the value is being converted to its minute equivalent.

Reply:
Maybe we can extend these constants into classes with getters such as `milliseconds`, `seconds`, `hours`, so we have for example, `Day.milliseconds`, `Day.seconds`, `Day.hours`, etc. Might prevent such a  mistake going further. Thoughts?

Reply:
I'm down for this change